lib/core/src/lib/progress/progress.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container [ngSwitch]="variant">
+    <ng-container *ngSwitchDefault>
+        <mat-progress-bar
+            [mode]="$any(mode)"
+            [value]="value"
+            [color]="color"
+            [attr.aria-label]="ariaLabel"
+            [attr.aria-hidden]="ariaHidden"
+            [attr.data-automation-id]="testId">
+        </mat-progress-bar>
+    </ng-container>
+
+    <ng-container *ngSwitchCase="'spinner'">
+        <mat-progress-spinner
```

Comment:
can we add `[diameter]` parameter to be able to change the size?
And may be it make sense to add any class or wrapper not to touch mat-selectors and change styles
For example, [here](https://github.com/Alfresco/alfresco-applications/pull/660/files#diff-a577b494dfb7e840dc813603dff14fd85f5c80cb5c704ce822e7ef8f5ec96f75R1) we have to use wrapper to make spinner in the center of the box

Reply:
We could probably investigate and do as new PRs and extensions. So far the diameter `looks` deprecated and might go away during our Angular upgrades

<img width="602" alt="Screenshot 2024-07-23 at 9 51 44 AM" src="https://github.com/user-attachments/assets/a21edee5-a862-4973-ac74-228471209b2c">